lib/cn/csched_sp3_work.h

```diff
@@ -31,7 +31,7 @@
 
 #define SP3_LCOMP_RUNLEN_MAX            (12u)
 #define SP3_LCOMP_SPLIT_PCT             (100u)
-#define SP3_LCOMP_SPLIT_KEYS            (128u) /* units of 4 million */
+#define SP3_LCOMP_SPLIT_KEYS            (64u) /* units of 4 million */
```

Comment:
Until we improve the rate at which we can kcompact/spill keys we need to lower the default number of keys to split on...  I am hoping Tristan's improvements to use bin-heap2 will improve this situation!